src/particles.cpp

```diff
@@ -251,6 +252,17 @@ void ParticleSpawner::step(float dtime, ClientEnvironment* env)
 {
 	m_time += dtime;
 
+	bool unloaded = false;
+	v3f attached_offset = v3f(0,0,0);
+	if (m_attached_id != 0) {
+		ClientActiveObject *attached = env->getActiveObject(m_attached_id);
+		if (attached) {
```

Comment:
I use attached in the next line, won't I have to call env->getActiveObject again?